tensorflow/tools/docker/Dockerfile.devel

```diff
@@ -38,6 +38,8 @@ RUN pip --no-cache-dir install \
         && \
     python -m ipykernel.kernelspec
 
+# RUN ln -s /usr/bin/python3 /usr/bin/python#
```

Comment:
Why is this line commented out?

Reply:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/docker/parameterized_docker_build.sh#L309

I believe your PR followed the same logic. We only want this for python3.

Reply:
Good point :)